meta/documents/user_guide/de/initial_setup.md

```diff
@@ -0,0 +1,187 @@
+Befolgen Sie die [Anweisungen für den Versand über Wayfair](wayfair_shipping.md) beschriebenen Verfahren, um eine ordnungsgemäße Integration mit Wayfair für den Versand von Bestellartikeln sicherzustellen.
+
+## 4. Verknüpfen Sie Wayfair-Bestellartikel mit Plentymarkets-Artikeln:
+Um eingehende Bestellungen von Wayfair ordnungsgemäß verarbeiten zu können, muss das Wayfair-Plugin die Lieferanten-Teilenummern aus Wayfairs Systemen mit einem bestimmten Feld von Artikelvariationen in Plentymarkets abgleichen. Standardmäßig geht das Wayfair-Plugin davon aus, dass die `Variationsnummer` **(nicht die ID der Varianten)** der Varianten eines Artikels in Plentymarkets mit der Teilenummer des Wayfair-Lieferanten übereinstimmt.
+
+![variation number field](../../../images/de/variation_number_field.png)
+
+Wenn die Lieferanten-Teilenummern für Ihre Organisation in einem alternativen Feld in Ihren Plentymarkets-ArtikelVarianten angezeigt werden, ändern Sie den Wert von [`Item Mapping Method`](settings_guide.md#item-mapping-method).
+
+## 5. Artikel auf Wayfair zum Verkauf anbieten
+Artikel, die Sie auf Wayfair verkaufen möchten, müssen in Plentymarkets als aktiv markiert werden. Der Benutzer von Plentymarkets kann auch festlegen, welche Artikel auf Wayfair zum Verkauf stehen. **Beachten Sie, dass der Lagerbestand und die bestellten Artikel auf der Ebene "Varianten" gesteuert werden.**
+
+Dieses Verfahren ist nur erforderlich, wenn [die Einstellung `Vollständigen Bestand an Wayfair senden?`](settings_guide.md#vollständigen-bestand-an-wayfair-senden) **deaktiviert** ist:
+
+1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Artikel` >> `Artikel bearbeiten`.
+
+2. Suchen Sie nach Artikeln und öffnen Sie sie
+
+3. **Klicken Sie für jedes Element** auf `Varianten` und öffnen Sie diese.
+
+4. **Für jede Variante**:
+
+    1. Stellen Sie auf der Registerkarte `Einstellungen` sicher, dass das Kontrollkästchen `Aktiv` im Abschnitt `Verfügbarkeit` aktiviert ist.
+
+        ![variation active](../../../images/de/variation_active_field.png)
+
+    2. Wenn [die Einstellung `Vollständigen Bestand an Wayfair senden?`](settings_guide.md#vollständigen-bestand-an-wayfair-senden) **deaktiviert** ist, wechseln Sie zur Registerkarte `Verfügbarkeit` der Variation und fügen Sie `Wayfair` zur Liste im Bereich `Märkte` hinzu.
+        1. Klicken Sie in den Bereich `Klicken, um Märkte auszuwählen`.
+
+        2. Scrollen Sie nach unten zum Eintrag `Wayfair`
+
+        3. Setzen Sie ein Häkchen neben `Wayfair`
+
+        4. Klicken Sie auf die Schaltfläche `+` ![add button](../../../images/common/button_plus.png) im Bereich "Märkte".
+
+             ![adding wayfair market to variation](../../../images/de/variation_wayfair_market.png)
+
+        5. Beachten Sie, dass in `Märkte` jetzt eine `Wayfair` -Reihe vorhanden ist
+
+            ![adding wayfair market to variation](../../../images/de/variation_wayfair_market_added.png)

```

Comment:
This screenshot does not have the Zwiesitzer Amsterdam at Dawn blurred like in the English screenshot